src/components/services/ServiceKonnector.jsx

```diff
@@ -6,26 +6,32 @@ import { getKonnectorIcon } from '../../lib/icons'
 
 const ENTER_KEY = 13
 
-const ServiceKonnector = ({ t, status, konnector, showKonnector }) => (
-  <li
-    onKeyDown={e => (e.keyCode === ENTER_KEY ? showKonnector(konnector) : null)}
-    onClick={() => showKonnector(konnector)}
-    tabIndex="0"
-    className="item-wrapper"
-  >
-    <header className="item-header">
-      <img
-        className="item-icon"
-        alt={t('connector.logo.alt', { name })}
-        src={getKonnectorIcon(konnector)}
-      />
-    </header>
-    <p className="item-title">{konnector.name}</p>
-    {konnector.category && (
-      <p className="item-subtitle">{t(`category.${konnector.category}`)}</p>
-    )}
-  </li>
-)
+const ServiceKonnector = ({ t, status, konnector, showKonnector }) => {
+  const categories = konnector.categories.map(category =>
```

Comment:
It feels weird to me to do this map here:
* `categories` could be `undefined`
* We then have a condition on `konnector.categories`

i suggest instead :
```js
const { categories } = konnector
```
and l. 29
```jsx
{categories && (<p className="item-subtitle">categories.map(c => t(`category.${c}`)).join(', ')}</p>)}
```

What do you think ?